app/mailers/organization_mailer.rb

```diff
@@ -1,10 +1,18 @@
+# organization.rb
+class Organization < ApplicationRecord
+  def send_submission_notification(partner)
```

Comment:
This doesn't need to be a method on the model class - it has nothing to do with the database, which is all the model should care about. The request code should call this itself.

Reply:
Also, you should not be re-opening / redefining model classes inside a mailer.